Signal/src/ViewControllers/ConversationView/ConversationViewModel.m

```diff
@@ -574,6 +585,32 @@ - (void)uiDatabaseDidUpdate:(NSNotification *)notification
                updatedNeighborItemSet:updatedNeighborItemSet];
 }
 
+// A simpler version of the update logic we use when
+// only transient items have changed.
+- (void)updateForTransientItems
```

Comment:
I'm not sure it's worth DRY'ing up the similar code between this and the method we use to respond to DB modification notifications.